modules/logging/filewriter.go

```diff
@@ -33,6 +34,43 @@ func init() {
 	caddy.RegisterModule(FileWriter{})
 }
 
+// FileMode is a string made of 1 to 4 octal digits representing
+// a numeric mode as specified with the `chmod` unix command.
+// `"0777"` and `"777"` are thus equivalent values.
```

Comment:
Will this ever be a limitation or problematic? (I guess, do people ever represent permissions using any other than the octal representation?) I just don't want to assume something we shouldn't be.

Reply:
I am not sure to understand the question.

The unix `chmod` command allows for both octal `0644` and symbolic `u=rw,o=r` files modes. So yes, people are using other than octal representation in their daily life with unix systems.

This PR only implements the octal one, but it does not prevent the symbolic one to be implemented later on if it turns out to be requested by users. I would be surprised, because most of server configuration files use the octal form ([logrotate for instance](https://manpages.debian.org/bookworm/logrotate/logrotate.8.en.html#create))

The symbolic representation will still need to be converted to the octal form as this is expected by `os.Chmod` Go function. So I do not see anything problematic at this stage.